### DISCUSSION TITLE
Registry API - Is it possible to  have labels and additional propertied in create API?

### DISCUSSION THREAD

**Discussion by ozarkblue** [Apicurio/apicurio-registry#2722] [Q&A] [answered]
I see these labels and propertied update option is there with Update Artifact API. Is it possible to do in same POST call for Create API ?
Trying to find a way so that can have these metadata as well to a new version or artifact  without making another update call. Is it possible?

**Comment by EricWittmann** [comment, marked as answer]
Currently only name, description, and version can be passed (as HTTP request headers) when creating a new artifact.  I'll open a GH issue to track adding labels and properties as well.  We might also extend the `application/create.extended+json` variant to allow this.

**Reply by EricWittmann** [reply]
https://github.com/Apicurio/apicurio-registry/issues/2742

**Comment by EricWittmann** [comment]
Update: this has been implemented in 3.0.
